kpi/views/environment.py

```diff
@@ -53,7 +63,12 @@ def get(self, request, *args, **kwargs):
                 processor = None
             value = getattr(constance.config, key)
             if processor:
-                value = processor(value)
+                try:
+                    value = processor(value)
+                except json.JSONDecodeError:
+                    logging.error(f'Environment.py: Could not decode `{key}`')
+                    continue
```

Comment:
This might be smart to keep to avoid bad settings basically crashing the entire server. However, it's also possible to validate the JSON when it's being saved from the admin interface. I don't remember how, exactly, but I did it for the custom metadata stuff :)

Reply:
Yep :-) https://github.com/kobotoolbox/kpi/blob/3563-disabled-date-support/kobo/settings/base.py#L249-L254 

Thank you for bushwacking